e2e-tests/development-runtime/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Unrelated to this PR specifically - do we need these separate .prettierrc files?

I think Prettier should automatically pick up the monorepo root prettierrc file: https://prettier.io/docs/en/configuration.html

Reply:
I guess this was added by installing the starter, maybe worth removing if we don't need it though.